tiatoolbox/tools/stainnorm/reinhard_colour_normaliser.py

```diff
@@ -0,0 +1,114 @@
+import numpy as np
+import cv2
+
+
+class ReinhardColourNormaliser:
+    """Normalize a patch colour to the target image using the method of:
+
+    Reinhard et al. "Color transfer between images." Computer graphics & applications.
+
```

Comment:
Need to define attributes for this class.

Reply:
Can you define what you mean by attributes here? Class attributes or instance attributes?

Reply:
Please consider the example at this [link](https://github.com/TIA-Lab/tiatoolbox/blob/a826f658dd02671631d83cfc42676b2858548fb5/tiatoolbox/dataloader/wsireader.py#L35)

Reply:
This isn't really clear by looking at that example. Are you referring to the variables defined as arguments in the __init__ or the instance variables defined in __init__ itself? Here, I didn't include attributes because __init__ only takes self as an argument.

Reply:
self.target_means = means
self.target_stds = stds